com.heroicgameslauncher.hgl.yml

```
@@ -0,0 +1,158 @@
+id: com.heroicgameslauncher.hgl
+sdk: org.freedesktop.Sdk
+runtime: org.freedesktop.Platform
+runtime-version: '21.08'
+base: org.electronjs.Electron2.BaseApp
+base-version: '21.08'
+command: heroic-run
+separate-locales: false
+
+sdk-extensions:
+  - org.freedesktop.Sdk.Compat.i386
+  - org.freedesktop.Sdk.Extension.toolchain-i386
+
+finish-args:
+  - --allow=devel
+  - --allow=multiarch
+  - --device=all
+  - --env=GST_PLUGIN_SYSTEM_PATH=/app/lib/gstreamer-1.0:/app/lib32/gstreamer-1.0:/usr/lib/extensions/gstreamer-1.0:/usr/lib/x86_64-linux-gnu/gstreamer-1.0:/usr/lib/i386-linux-gnu/gstreamer-1.0
+  - --filesystem=~/.local/share/lutris:ro
```

Unless you hardcode the `~/.local/share/lutris` path in the HGL app (i.e. not XDG-compliant),
```suggestion
  - --filesystem=xdg-data/lutris:ro
```

---

fixed